pydoctor/sphinx_ext/build_apidocs.py

```diff
@@ -42,11 +43,29 @@ def on_build_finished(app: Sphinx, exception: Exception) -> None:
         'outdir': app.outdir,
         }
 
+    runs = config.pydoctor_args
+
+    if not isinstance(runs, Mapping):
+        # We have a single pydoctor call
+        runs = {'main': runs}
+
+    for key, value in runs.items():
+        _run_pydoctor(key, value, placeholders.copy())
+
+
+def _run_pydoctor(name: str, arguments: List[str], placeholders: Mapping[str, str]) -> None:
```

Comment:
Do you need to modify the arguments? If not, it's better to use an immutable type such as `Sequence` or `Iterable`.